runtime/src/snapshot_package.rs

```diff
@@ -16,14 +13,73 @@ use solana_sdk::hash::Hash;
 use std::{
     fs,
     path::{Path, PathBuf},
-    sync::mpsc::{Receiver, SendError, Sender},
+    sync::{
+        mpsc::{Receiver, SendError, Sender},
+        Arc, Mutex,
+    },
 };
 use tempfile::TempDir;
 
+/// The sender side of the AccountsPackage channel, used by AccountsBackgroundService
 pub type AccountsPackageSender = Sender<AccountsPackagePre>;
+
+/// The receiver side of the AccountsPackage channel, used by AccountsHashVerifier
 pub type AccountsPackageReceiver = Receiver<AccountsPackagePre>;
+
+/// The error type when sending an AccountsPackage over the channel fails
 pub type AccountsPackageSendError = SendError<AccountsPackagePre>;
 
+/// The PendingSnapshotPackage passes a SnapshotPackage from AccountsHashVerifier to SnapshotPackagerService for archiving
+pub type PendingSnapshotPackage = Arc<Mutex<Option<SnapshotPackage>>>;
```

Comment:
Does this make more sense to leave in `snapshot_packager_service.rs` since it defines the interface between  the `SnapshotPackagerService` itself and `AccountsHashVerifier`?

Reply:
Yeah, I was wondering about this myself... I moved it because `snapshot_package.rs` defines the interface types between `AccountsBackgroundService` and `AccountsHashVerifier`, so I was thinking it'd be nice to have it all in one spot. Should I move `PendingSnapshotPackage` back?

https://github.com/solana-labs/solana/blob/9dfeee2993312accb61b3d48270671de2774eb81/runtime/src/snapshot_package.rs#L23-L25

Reply:
ah kk, I think that's fine then, prefer to keep them all in one spot